tests/test_exptime.py

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+import os
+import sys
+import unittest
+
+import requests
+
+pkg_root = os.path.abspath(os.path.join(os.path.dirname(__file__), '..'))  # noqa
```

Comment:
what does `# noqa` mean?

Reply:
one is not supposed to have code statements before all the imports, so this is flagged as a lint error.

Reply:
@Bento007, I'm not sure if you were asking about what `#noqa` means or why the tag is on this line. The noqa tag suppresses linter (flake8) errors on this line, which would normally cause tests to fail.